### ISSUE TITLE
[Accessibility] Slides Page: Select element does not have accessible name

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#970]
**Description**
`<select>` element does not have an accessible name

**Screenshots**
![image](https://github.com/camicroscope/caMicroscope/assets/122698422/f6e0d9f0-aa3b-4703-8ef4-d2bf8320d7d1)

**Proposed Fix**
Add an `aria-labelledby` attribute to the `<select>` element
